test/analytics/fixtures/plugins/analytics_ftr_helpers/public/plugin.test.ts

```diff
@@ -123,6 +123,7 @@ describe('AnalyticsFTRHelpers', () => {
         window.__analytics_ftr_helpers__.getEvents(2, {
           eventTypes: [event.event_type],
           fromTimestamp: '2022-03-10T00:00:00.000Z',
+          withTimeoutMs: 100,
```

Comment:
you could have changed the number to 1 as well. the test would finish earlier :)

Reply:
I meant the number of expected events (replacing `2` with `1`) so there is no need for the `withTimeoutMs` 😇 

But that's fine as is :)

Reply:
I actually think it's best to fetch for more events, otherwise you would not catch cases when a double event is fired (like it does right now for the maps embeddable)

Reply:
makes sense! thanks for explaining!